tests/integration/lineage.js

```diff
@@ -0,0 +1,481 @@
+const { expect } = require('chai');
+const sinon = require('sinon').sandbox.create();
+const lineageFactory = require('../../shared-libs/lineage/src/lineage');
```

Comment:
Not sure how I feel about this and whether it would be better to install `lineage` as a package at the root level of the repo

Reply:
Seems clear enough like this to me.